src/QuoteReducer.js

```diff
@@ -0,0 +1,47 @@
+import { 
```

Comment:
In case where You start adding saga this file composition wont work, nextQuote would go into saga, When sagas exist it would be good not to do any operations inside reducers, only mergening received data into state. We can say that reducers become useless when saga starts knocking on the door and saying "I will take it for You from now on". But of course, there is no rule that would disable You from doing that, would not damage the functionality.

There is a handleActions method that would help You to handle specific actions without switching, You can check it here:  https://github.com/Healthcasts/healthcasts-docdx/blob/master/react/src/data/answers/reducer.js

Its also not important from performance view

Reply:
Search for immutable npm package